server/statistics/hot_cache_task.go

```diff
@@ -79,33 +79,33 @@ func (t *checkExpiredTask) taskType() flowItemTaskKind {
 func (t *checkExpiredTask) runTask(cache *hotPeerCache) {
 	expiredStats := cache.collectExpiredItems(t.region)
 	for _, stat := range expiredStats {
-		updateStat(cache, stat)
+		cache.updateStat(stat)
 	}
 }
 
-type collectUnReportedPeerTask struct {
+type checkColdPeerTask struct {
 	storeID  uint64
 	regions  map[uint64]*core.RegionInfo
 	interval uint64
 }
 
-// NewCollectUnReportedPeerTask creates task to collect unreported peers
-func NewCollectUnReportedPeerTask(storeID uint64, regions map[uint64]*core.RegionInfo, interval uint64) flowItemTask {
-	return &collectUnReportedPeerTask{
+// NewCheckColdPeerTask creates task to check cold peers
+func NewCheckColdPeerTask(storeID uint64, regions map[uint64]*core.RegionInfo, interval uint64) flowItemTask {
```

Comment:
Do we have to modify it? I think the previous name was better

Reply:
@Yisaer WDYT?

Reply:
how about `CheckSilentPeerTask`

Reply:
It's worse I think. Actually, the regions here are reported instead of unreported. So `CollectUnReportedPeer` is not accurate.